swayfmt/src/utils/language/expr/mod.rs

```diff
@@ -282,6 +282,7 @@ impl Format for Expr {
                     |formatter| -> Result<(), FormatterError> {
                         write!(formatted_code, "{} ", for_token.span().as_str())?;
                         value_pattern.format(formatted_code, formatter)?;
+                        write!(formatted_code, " ")?;
                         write!(formatted_code, "{} ", in_token.span().as_str())?;
```

Comment:
Sure this can be this:
```suggestion
                        write!(formatted_code, " {} ", in_token.span().as_str())?;
```